admin/class-ajax.php

```
@@ -835,8 +837,72 @@
+                       } elseif ( ai_services()->is_service_available( 'google' ) ) {
+                               $service = ai_services()->get_available_service( 'google' );
+                       } else {
+                               $error = new \WP_Error( '-5', __( 'No AI service available', 'accessibility-checker' ) );
+                               wp_send_json_error( $error );
+                       }
+
+                       $candidates = $service
+                               ->get_model(
+                                       [
+                                               'feature'      => 'edac-simplified-summary',
+                                               'capabilities' => [ \Felix_Arntz\AI_Services\Services\API\Enums\AI_Capability::TEXT_GENERATION ],
```

PHPStan (2.1.17)

Access to constant TEXT_GENERATION on an unknown class Felix_Arntz\AI_Services\Services\API\Enums\AI_Capability.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)